base/applications/network/ping/lang/zh-CN.rc

```diff
@@ -1,66 +1,66 @@
 /*
  * FILE:        base/applications/network/ping/lang/zh-CN.rc
  * PURPOSE:     Chinese (Simplified) translations for ReactOS Ping Command
- * TRANSLATORS:
+ * TRANSLATORS: 2018 Luo Yufan (njlyf2011@hotmail.com)
  */
 
 LANGUAGE LANG_CHINESE, SUBLANG_CHINESE_SIMPLIFIED
 
 STRINGTABLE
 BEGIN
     IDS_USAGE "\n\
-Usage: ping [-t] [-a] [-n count] [-l size] [-f] [-i TTL] [-v TOS]\n\
+用法: ping [-t] [-a] [-n count] [-l size] [-f] [-i TTL] [-v TOS]\n\
             [-w timeout] [-4] [-6] target\n\
 \n\
 Options:\n\
-    -t          Ping the specified host until stopped.\n\
-                To see statistics and continue - type Control-Break;\n\
-                To stop - type Control-C.\n\
-    -a          Resolve addresses to hostnames.\n\
-    -n count    Number of echo requests to send.\n\
-    -l size     Send buffer size.\n\
-    -f          Set Don't Fragment flag in packet (IPv4-only).\n\
-    -i TTL      Time To Live.\n\
-    -v TOS      Type Of Service (IPv4-only. This setting has been deprecated\n\
-                and has no effect on the type of service field in the IP\n\
-                Header).\n\
-    -w timeout  Timeout in milliseconds to wait for each reply.\n\
-    -4          Force using IPv4.\n\
-    -6          Force using IPv6.\n\
+    -t          Ping 指定的主机，直到停止。\n\
+                若要查看统计信息并继续操作，请键入 Ctrl+Break；\n\
+                若要停止，请键入 Ctrl+C。\n\
+    -a          将地址解析为主机名。\n\
+    -n count    要发送的回显请求数。\n\
+    -l size     发送缓冲区大小。\n\
+    -f          在数据包中设置“不分段”标记(仅适用于 IPv4)。\n\
+    -i TTL      生存时间。\n\
+    -v TOS      服务类型(仅适用于 IPv4。该设置已被弃用，\n\
+                对 IP 标头中的服务类型字段没有任何\n\
+                影响)。\n\
+    -w timeout  等待每次回复的超时时间(毫秒)。\n\
+    -4          强制使用 IPv4。\n\
+    -6          强制使用 IPv6。\n\
 \n"
 
     IDS_CTRL_BREAK "Control-Break\n"
     IDS_CTRL_C "Control-C\n"
-    IDS_NO_RESOURCES "Not enough resources available.\n"
-    IDS_MISSING_ADDRESS "IP address must be specified.\n"
-    IDS_MISSING_VALUE "Value must be supplied for option %s.\n"
-    IDS_BAD_OPTION "Bad option %s.\n"
-    IDS_BAD_PARAMETER "Bad parameter %s.\n"
+    IDS_NO_RESOURCES "没有足够的可用资源。\n"
+    IDS_MISSING_ADDRESS "必须指定 IP 地址\n"
+    IDS_MISSING_VALUE "必须为选项 %s 提供值。\n"
+    IDS_BAD_OPTION "选项 %s 不正确。\n"
+    IDS_BAD_PARAMETER "错误的参数 %s.\n"
     IDS_BAD_VALUE "Bad value for option %s, valid range is from %u to %u.\n"
```

Comment:
Some strings remain untranslated, is it because your translation update is still work-in-progress?